samples/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <bluetooth/bluetooth.h>
+#include <shell/shell.h>
+
+#define MODULE ble_bond
+#include "module_state_event.h"
+#include "button_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_BOND_LOG_LEVEL);
+
+static bool bonds_initialized;
+static bool remove;
+
+static int bonds_remove(void)
```

Comment:
can we rename it to `remove_bonds`? It seems more natural and there is only one function so `bond_` is not really a prefix.

Reply:
I prefer `noun_verb()` as in `peer_info_show()` or `cmd_bonds_remove()`. I would like to leave it like this, otherwise we could go through all application and unify it.